content/news/vcmi-0.100-release.md

```diff
@@ -0,0 +1,35 @@
++++
+categories = ["Releases", "Stable"]
+date = "2022-09-10"
+description = ""
+featured = "vcminewterrain.jpg"
+featuredalt = ""
+featuredpath = "date"
+linktitle = ""
+title = "VCMI 0.100 released"
+type = "post"
+forumurl = "https://forum.vcmi.eu/"
+
++++
+
+## We're happy to announce new release!
+
+Few years passed after we released VCMI 0.99 and now it's time for a new release.
+Tons of changes, improvements, and fixes have been introduces during that time, it's even hard to recall all of them.
+However we'd like to highlight several major changes.
+
+## Artificial intellengence
+You are welcome to try brand new AI algorithm called Nullkiller or you can play with classic VCAI computer player.
```

Comment:
```suggestion
You are welcome to try an alternative AI algorithm called Nullkiller which shows itself well on small to medium maps. For larger maps you should better go with classic VCAI computer player which also became much smarter ...
```

Reply:
`shows itself well on small to medium maps` in the sentence `to` seems strange. If there is no sizes between small and medium, I would write `shows itself well on small and medium maps`. If there are sizes between them, I would write something like `shows itself well on maps up to medium size`.

Reply:
Maybe "or"

Reply:
shows itself well on small and medium maps - this sounds better, thank you